etc/deprecated_rules.json

```diff
@@ -129,6 +129,11 @@
     "rule_name": "PPTP (Point to Point Tunneling Protocol) Activity",
     "stack_version": "7.14.0"
   },
+  "dc672cb7-d5df-4d1f-a6d7-0841b1caafb9": {
```

Comment:
The deprecation was not backported in order to deprecate it only as of 8.0+. This means that once this lock backports, the rule will exist in an undeprecated state in 7.13-7.16, but reflecting as deprecated in the `deprecated_rules` file. It also still exists in the `version.lock` since all deprecated rules remain unchanged there.

Need to think through the implications of this ...

Reply:
* the rule_loader [code](https://github.com/elastic/detection-rules/blob/84d55c829dcaa15b427033894f22bb397557ddd1/detection_rules/rule_loader.py#L222) loads rules into the deprecated collection based on the maturity of the rule itself. This would load the rule into the normal (non-deprecated) collection in the previous branches, as expected.
* [VersionLock](https://github.com/elastic/detection-rules/blob/84d55c829dcaa15b427033894f22bb397557ddd1/detection_rules/version_lock.py#L93-L146) was reviewed and is ok - it will not be detected as a new deprecation and only newly deprecated rules result in changes to the version.lock and deprecated_rules files
* [Package](https://github.com/elastic/detection-rules/blob/84d55c829dcaa15b427033894f22bb397557ddd1/detection_rules/packaging.py#L300) documents newly deprecated rules, so this is ok as well

cc: @DefSecSentinel @threat-punter